roles/openshift_prometheus/defaults/main.yaml

```diff
@@ -6,11 +6,15 @@ openshift_prometheus_namespace: prometheus
 
 openshift_prometheus_node_selector: {"region":"infra"}
 
-# images
-openshift_prometheus_image_proxy: "openshift/oauth-proxy:v1.0.0"
-openshift_prometheus_image_prometheus: "openshift/prometheus:v2.0.0-dev.3"
-openshift_prometheus_image_alertmanager: "openshift/prometheus-alertmanager:v0.9.1"
-openshift_prometheus_image_alertbuffer: "openshift/prometheus-alert-buffer:v0.0.2"
+# image defaults
```

Comment:
This will make our deployment more complicated than before, I think you may misunderstood my opinions
I think we could use  only one "****_image_prefix" to stands for all images prefix, then we don't need to set openshift_prometheus_image_prefix, openshift_prometheus_proxy_image_prefix, openshift_prometheus_alertmanager_image_prefix, openshift_prometheus_alertbuffer_image_prefix, all prometheus component will share one same image prefix.

It is the same for **image_version.

Take Logging  for example, please refer to [1]

[1] https://privatebin-it-iso.int.open.paas.redhat.com/?d217713dce3e9dc5#BMi7r06l/mcICwo9dIUAPLUiez/okcaGTanfDjH2V/I=

Reply:
I understand,
@sdodson said he will take care of that here: https://github.com/openshift/openshift-ansible/pull/5787#issuecomment-339011263

Reply:
#5969 should help